packages/chakra-ui/src/Widgets/Widgets.ts

```diff
@@ -22,6 +23,7 @@ export function generateWidgets<
     RadioWidget,
     RangeWidget,
     SelectWidget,
+    NativeSelectWidget,
```

Comment:
Is the goal of the `NativeSelectWidget` for someone to override it via a `uiSchema` widget declaration?